pkg/tasks/nodeexporter_dashboard.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 The Cluster Monitoring Operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tasks
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openshift/cluster-monitoring-operator/pkg/client"
+	"github.com/openshift/cluster-monitoring-operator/pkg/manifests"
+)
+
+type NodeExporterDashboardTask struct {
+	client  *client.Client
+	factory *manifests.Factory
+}
+
+func NewNodeExporterDashboardTask(client *client.Client, factory *manifests.Factory) *NodeExporterDashboardTask {
+	return &NodeExporterDashboardTask{
+		client:  client,
+		factory: factory,
+	}
+}
+
+func (t *NodeExporterDashboardTask) Run(ctx context.Context) error {
```

Comment:
I think this could be part of the existing node exporter task.

Reply:
or even have it deployed by CVO as for the others https://github.com/openshift/cluster-monitoring-operator/blob/master/manifests/0000_90_cluster-monitoring-operator_02-dashboards.yaml if we agree on having it on all clusters.

Reply:
@machine424 , i put it in a different task as a preparation for adding metrics existence check, although i am not sure if it will work: if the certificate used by  thanos-querier is stored in the secret, and that secret is deployed by the CVO in a dedicated task, then it kind of "chicken-egg" issue.